### ISSUE TITLE
Trained model file is very larger

### ISSUE THREAD

**Issue by kyodaisuki** [mrastegari/XNOR-Net#16]
Hi, I had trained alexnet_XNOR by myself.
(I have a smaller GPU, so the flag which i use "-nGPU 1 -batchSize 400"
It seems converged and every thing is fine. (42.1% Top-1 accuracy on validation dataset
But the question is that the saved model occupy so many disks space ..
the model_epoch.t7 need 4.8 GB
the optimState_epoch.t7 need 0.75 GB
and we need 55 epochs !!

I don't know why your download pre-train model need only 0.5 GB
Could you help me to save my disk space ?
